gdcdictionary/schemas/medical_history.yaml

```diff
@@ -1117,12 +1117,147 @@ properties:
       LVEF by echo (closet to enrollment)
     type: number
   
-  ef_ever_50:
```

Comment:
What is going on here, why is ef_ever_50 being deleted?

Reply:
resolved.  Accidently, deleted property, but added back.